tests/Doctrine/Tests/ORM/Functional/Ticket/Issue7052Test.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Tests\Models\CMS\CmsEmployee;
+
+/**
+ * @group 7052
+ */
+class Issue7052Test extends \Doctrine\Tests\OrmFunctionalTestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->_schemaTool->createSchema(array(
+            $this->_em->getClassMetadata(__NAMESPACE__ . '\\Issue7052Child'),
+            $this->_em->getClassMetadata(__NAMESPACE__ . '\\Issue7052Parent'),
+        ));
+    }
+
+    public function testIssue()
+    {
+        $parent       = new Issue7052Parent();
+        $parent->name = "parent test";
+
+        $childA         = new Issue7052Child();
+        $childA->parent = $parent;
+
+        $childB         = new Issue7052Child();
+        $childB->parent = $parent;
+
+        $this->_em->persist($parent);
+        $this->_em->persist($childA);
+        $this->_em->persist($childB);
+        $this->_em->flush();
+        $this->_em->clear();
+
+        $childAFromDb = $this->_em->find('Doctrine\Tests\ORM\Functional\Ticket\Issue7052Child', $childA->id);
+
+        $this->_em->clear();
+
+        $childBFromDb = $this->_em->find('Doctrine\Tests\ORM\Functional\Ticket\Issue7052Child', $childB->id);
+
+        $parentFromChildB = $childBFromDb->parent;
+
+        self::assertNotSame($childA, $childAFromDb);
+        self::assertNotNull($childAFromDb->parent->name, "Unable to get parent name on second loaded child after EM cleared.");
```

Comment:
I don't think this assertion is valid.
1. You loaded `$childAFromDb` from DB.
2. You called `EntityManager::clear()`, making the obtained entity unmanaged.
3. You assert that lazy association works on an unmanaged entity.

This test is also failing in ORM 2.5.x.

Reply:
Also, if this works in master, it's rather a bug there, since once unmanaged, entities/proxies should be ignored by ORM.

Reply:
@Ocramius @lcobucci Please verify the assumptions above. Removing from 2.6 milestone for now.